### PR TITLE
Metadata updates

### PR DESCRIPTION
**Pull request by brforest**
- Add correct last_modified field in metadata file
- Remove unused fields from metadata file
- Add a README at recommended_scripts directory level
  - Contains short descriptions pulled from "Description" field in each script's Manifest object
- Move all scripts to their own directory
- Update metadata generator to handle sub-directories
- Add READMEs at individual script level
  - Contains long descriptions pulled from LONG_DESCRIPTION constant in each script
  - Added LONG_DESCRIPTION for two scripts as a sample

### CODE REVIEW

File: recommended_scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+Monitor rate of decrease of routes
+
+### connectivity_monitor:
+This script monitors the reachability between two devices given the IP-SLA session.The IP-SLA session has to be configured in the switch before using this script to monitor the connectivity/reachability between two devices.
+
+### power_supply_monitors:
+System Power Supply monitoring agent
+
+### configuration_change_tftp:
+Agent to copy the running configuration to a TFTP serverwhenever a configuration change occurs.
```
Comment (alagoutte) [inline recommended_scripts/README.md:27]: Missing space (serverwhenever)

File: recommended_scripts/README.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
+# Recommended NAE Scripts
+
+These are the scripts recommended by HPE Aruba Networks for customers to use on their switches. These scripts have been tested and determined to add value.  Below is a list of all of the scripts with a short description. You can click on each script directory to see all available versions (different versions have different firwmare support), as well as a README with a more detailed description of each script.  All of these scripts can also be seen in the switch Web UI NAE script portal.
+
+### copp:
+This script monitors the CoPP (COntrol Plane Policing) policy configured on the switch. Traffic destined to the switch dropped by CoPP due to a high traffic rate is counted as traffic dropped and traffic allowed to reach the switch control plane will be counted as traffic allowed. Alerts are generated when these rates exceed their respective thresholds. The thresholds are based on the the CoPP Class rates and the values entered by the user as parameters.
+
+### lag_health_monitor:
```
Comment (alagoutte) [inline recommended_scripts/README.md:8]: sort by Alpha ?

File: recommended_scripts/client_services_health/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
+## Summary
+
+Client Service Health for Radius, DNS and DHCP
```
Comment (alagoutte) [inline recommended_scripts/client_services_health/README.md:3]: RADIUS

File: recommended_scripts/configuration_change_email/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+## Summary
+
+Agent to alert user when configuration changes and show the system info and diff of configuration changes. Anemail notification with the diff is sent whenever the configuration changes.
```
Comment (alagoutte) [inline recommended_scripts/configuration_change_email/README.md:3]: anemail -> An email

File: recommended_scripts/ipsla/README.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+## Summary
+
+Monitor particular value/aggregate value of aSLA test and specify shell command to run as action
```
Comment (alagoutte) [inline recommended_scripts/ipsla/README.md:3]: a SLA

File: recommended_scripts/ipsla/ipsla.py [hunk 6]
```diff
@@ -18,7 +18,7 @@
 import time
 
 Manifest = {
-    'Name': 'ipsla_threshold',
+    'Name': 'ipsla',
     'Description': 'Monitor particular value/aggregate value of a'
```
Comment (alagoutte) [inline recommended_scripts/ipsla/ipsla.py:22]: missing also space after a

File: recommended_scripts/vsx_health_monitor/README.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+VSX Health monitoring
+
+## Supported Software Versions
+
+Script Version 4.0: ArubaOS-CX 10.13.1000 Minimum
+
+## Supported Platforms
+
+Script Version 4.0: 64xx, 8320, 8325, 8400
```
Comment (alagoutte) [inline recommended_scripts/vsx_health_monitor/README.md:11]: No support of 8100 / 8360 ?

File: recommended_scripts/README.md (hunk 2 above)
Comment (brforest) [reply recommended_scripts/README.md:8]: Good idea - I will implement that.

File: recommended_scripts/vsx_health_monitor/README.md (hunk 7 above)
Comment (brforest) [reply recommended_scripts/vsx_health_monitor/README.md:11]: As far as I know, no.  I will check with NTL.

Comment (brforest) [reply recommended_scripts/vsx_health_monitor/README.md:11]: I checked with NTL and these platforms will be added, so I will add them to the list.

**Pull request merged**